CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # API Mediation Layer Changelog
 
 All notable changes to the Zowe API Mediation Layer package will be documented in this file.
+## `1.19.0`
+
+- Feature: Configure the connection limit for the GW [#843](https://github.com/zowe/api-layer/issues/843)
+- Feature: Minimize size of API Mediation Layer [#909](https://github.com/zowe/api-layer/issues/909)
+- Feature: Allow removal or configuration of a service as a catalog [#727](https://github.com/zowe/api-layer/issues/727)
+- Bugfix: API ID is not sent to Eureka in metadata by Java enabler [#991](https://github.com/zowe/api-layer/issues/991)
+- Feature: Improve connection limits [#987](https://github.com/zowe/api-layer/issues/987)
+- Feature: SAF authorization checking code [#993](https://github.com/zowe/api-layer/issues/993)
+- Feature: Enhance connection limit log messages [#987](https://github.com/zowe/api-layer/issues/987)
```

Comment:
Does enhancing the connection limit of log messages produce any of the following benefits? If so which ones? Is there one or two that are most significant?

- Documents the actual connection limit or state actual limits are based on resources assigned to the job.
- Documents why the connection limits exist and what happens when the limits are hit. 
- Documents symptoms that show the gateway is being stressed with too many connections. 
- Documents what error messages may be shown if there are too many connections.
- Logs a warning if the connection limit is reached, or if the APIML is experiencing stress from too many connections.
- Throws an error if the connection limit is too small.

Reply:
Please describe the benefit to the end user.

Reply:
I think that these are the main benefits:
Documents symptoms that show the gateway is being stressed with too many connections.
Documents what error messages may be shown if there are too many connections.

But @CarsonCook will know more